dev/PagerControl/InteractionTests/PagerControlTestBase.cs

```diff
@@ -30,9 +30,12 @@ public class PagerControlTestBase
 
         protected void SelectValueInPagerComboBox(int index)
         {
+            // Open ComboBox.
             InputHelper.LeftClick(elements.GetPagerComboBox());
-            InputHelper.LeftClick(elements.GetPagerComboBox().Children[index]);
             Wait.ForIdle();
+            // Index is the actual PagerControl SelectedIndex, so we need to add 1 here 
+            // to result in the user friendly name being displayed in the ComboBox.
+            new ComboBox(elements.GetPagerComboBox()).SelectItemByName((index + 1).ToString());
```

Comment:
Can we change GetPagerComboBox to return a ComboBox element instead of a UIObject?

Reply:
Good idea, updated that now.